app/models/agents/weather_agent.rb

```diff
@@ -135,98 +108,82 @@ def validate_location
     end
 
     def validate_options
-      errors.add(:base, "service must be set to 'darksky' or 'wunderground'") unless wunderground? || dark_sky?
```

Comment:
I think it would be nice to show an error message that the wunderground API shut down if an Agent is still configured as `wunderground?`.

Reply:
I added a check to display an error message if Weather Underground is still configured.